src/cargo/ops/cargo_rustc/job_queue.rs

```diff
@@ -76,7 +78,7 @@ impl<'a> JobState<'a> {
 }
 
 impl<'a> JobQueue<'a> {
-    pub fn new<'cfg>(cx: &Context<'a, 'cfg>) -> JobQueue<'a> {
+    pub fn new<'cfg>(cx: &Context<'a, 'cfg>, is_release: bool, is_doc_all: bool) -> JobQueue<'a> {
```

Comment:
I think these booleans are actually accessible through [`cx.build_config`](https://github.com/rust-lang/cargo/blob/master/src/cargo/ops/cargo_rustc/context.rs#L39), so perhaps they don't need to be passed in?